src/test/java/uk/gov/hmcts/sptribs/judicialrefdata/JudicialServiceTest.java

```diff
@@ -49,7 +49,6 @@ class JudicialServiceTest {
 
     @Test
     void shouldPopulateUserDynamicList() {
-        //Given
```

Comment:
We make sure all the null and empty values are handled properly in JudicialService and IdamService by adding specific test cases to JudicialServiceTest and IdamServiceTest files. And also make sure all the flow are covered at least by one test case.